src/focus.service.ts

```diff
@@ -362,46 +332,35 @@ export class FocusService {
       return;
     }
 
-    this.rescroll(next, scrollSpeed, this.root);
-
     const attached = selected && isNodeAttached(selected, this.root);
 
-    this.parents = [];
-
     if (attached) {
       // Find the common ancestor of the next and currently selected element.
       // Remove selected classes in the current subtree, and add selected
       // classes in the other subtree. Trigger focus changes on every
       // element that we touch.
-      selected.blur();
-
       const common = getCommonAncestor(next, selected);
-      selected.classList.remove(cssClass.direct);
```

Comment:
It sounds like the underlying goal is to make focus states on xbox look differently than they do on websites. If that's the case, I believe there may be simpler ways to go about doing that, like using our existing style loaders.